posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -77,6 +77,7 @@ def to_main_query(self) -> ast.SelectQuery:
 )
 GROUP BY "context.columns.breakdown_value"
 ORDER BY "context.columns.visitors" DESC,
+"context.columns.views" DESC,
```

Comment:
@robbie-c Do you see any problem with this new clause here? It's helpful when different fields have the same number of visitors, but a different number of views. It's more helpful during tests than on production data. Anything against the change?